src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -56,6 +57,14 @@ const PageContent = ( {
 export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProps ) {
 	const { locale } = useI18nData();
 	const whatsNewPages: WhatsNewPage[] = [
+		{
+			image: preferredAppsIllustration,
+			title: __( 'Choose your preferred apps' ),
+			description: __(
+				'Select your preferred code editor and terminal apps in the redesigned Settings modal.'
```

Comment:
```suggestion
				'Select your preferred code editor and terminal app in the redesigned Settings modal.'
```

I guess it could be just singular considering that we are selecting just one of each 😅 Not a strong preference though

Reply:
Yes, this version seems more correct to me as well, thanks for the suggestion!

Reply:
Done in 6f222f73177f3ee8af4da1b12472a2ea178183fe